platforms/linux-riscv64/Dockerfile

```
@@ -36,6 +36,7 @@ RUN \
     --no-modify-path \
     --profile minimal \
     && \
+  rustup default nightly && \
   rustup target add riscv64gc-unknown-linux-gnu && \
```

```suggestion
    --profile minimal \
    --target riscv64gc-unknown-linux-gnu \
    --default-toolchain nightly \
    && \
```
(I also changed it to use `--target ...`  instead of `rustup target add ...`)